lib/teamsnap.rb

```diff
@@ -34,7 +34,7 @@ def init(opts = {})
       self.root_client = TeamSnap::Client.new(:token => token)
 
       ##   Make the apiv3 root call. collection is parsed JSON
-      collection = TeamSnap.run(root_client, :get, "/", {})
+      collection = TeamSnap.run(root_client, :get, self.url, {})
```

Comment:
It seems to be working with the `/v3` but I will double check. What kind of error where you seeing?